doc/manuals/admin/database_model.md

```diff
@@ -86,55 +86,55 @@ Fields:
 Example document:
 
 ```
- {
```

Comment:
My IDE has ruin the whole formatting (only one line was touched). I will fix it.